receiver/zookeeperreceiver/metrics.go

```diff
@@ -0,0 +1,81 @@
+// Copyright 2020, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package zookeeperreceiver
+
+import (
+	"go.opentelemetry.io/collector/consumer/pdata"
+
+	"github.com/open-telemetry/opentelemetry-collector-contrib/receiver/zookeeperreceiver/internal/metadata"
+)
+
+// Constants to define entries in the output of "mntr" command.
+const (
+	avgLatencyMetricKey              = "zk_avg_latency"
+	maxLatencyMetricKey              = "zk_max_latency"
+	minLatencyMetricKey              = "zk_min_latency"
+	numAliveConnectionsMetricKey     = "zk_num_alive_connections"
+	outstandingRequestsMetricKey     = "zk_outstanding_requests"
+	zNodeCountMetricKey              = "zk_znode_count"
+	watchCountMetricKey              = "zk_watch_count"
+	ephemeralsCountMetricKey         = "zk_ephemerals_count"
+	approximateDataSizeMetricKey     = "zk_approximate_data_size"
+	openFileDescriptorCountMetricKey = "zk_open_file_descriptor_count"
+	maxFileDescriptorCountMetricKey  = "zk_max_file_descriptor_count"
+
+	followersMetricKey       = "zk_followers"
+	syncedFollowersMetricKey = "zk_synced_followers"
+	pendingSyncsMetricKey    = "zk_pending_syncs"
+
+	serverStateKey = "zk_server_state"
+	zkVersionKey   = "zk_version"
+
```

Comment:
What is the plan for the metrics exposed by Zookeeper that are not currently in this list?

Reply:
I will be adding 3 more metrics in #1404. I believe that would get us an exhaustive list of metrics exposed by Zookeeper up until version 3.6.0. Version 3.6.0 and above exposes more metrics via prometheus (this was added recently) and users should use the prometheus exporter for those versions Zookeeper.